ipv8/attestation/trustchain/block.py

```diff
@@ -26,8 +27,73 @@ class TrustChainBlock(object):
     """
     Container for TrustChain block information
     """
+    Data = namedtuple('Data', ['type',
+                               'transaction',
+                               'public_key',
+                               'sequence_number',
+                               'link_public_key',
+                               'link_sequence_number',
+                               'previous_hash',
+                               'signature',
+                               'timestamp',
+                               'insert_time'])
+    """
+    Data struct to initialize a TrustChainBlock.
+
+
+        **[0] type:** The block type name, as utf-8 string or utf-8 encoded bytes.
+
+        *> type:* str or bytes
+
+        **[1] transaction:** Metadata dictionary, consisting of binary type strings.
+
+        *> type:* dict
+
+        **[2] public_key:** The serialized public key of the initiator of this block (binary data).
+
+        *> type:* bytes (Py3) or str (Py2)
+
+        **[3] sequence_number:** The height of this block in the chain of the intiator of this block.
+
+        *> type:* int
+
+        **[4] link_public_key:** The serialized public key of the counterparty of this block (binary data).
+
+        *> type:* bytes (Py3) or str (Py2)
+
+        **[5] link_sequence_number:** The height of this block in the chain of the intiator of this block,
+        or 0 if unknown.
+
+        *> type:* int
+
+        **[6] previous_hash:** The hash of the previous block in the chain of the initiator of this block (binary data).
+
+        *> type:* bytes (Py3) or str (Py2)
+
+        **[7] signature:** The signature of the initiator of this block for this block (binary data).
+
+        *> type:* bytes (Py3) or str (Py2)
+
+        **[8] timestamp:** The time in milliseconds since the UNIX epoch that this block was created
+        (according to the initiator).
+
+        *> type:* int
+
+        **[9] insert_time:** The time in milliseconds since the UNIX epoch that this block was inserted into the local
```

Comment:
"The time in milliseconds since the UNIX epoch ~that~ when this block was inserted". Also, the same for `[8]`